### DISCUSSION TITLE
NFT: support standard License field for Copyright/Usage Rights info.

### DISCUSSION THREAD

**Discussion by oO** [deso-protocol/dips#186] [Medium CIPs]
Right now, buyers and sellers have no way to easily know what rights they are getting when buying an NFT. There should be a standard way for creators to either pick from a set of existing licenses, or create/link to their own license. This is going to become more and more important as use cases of NFTs on DeSo evolve.

The license should be a link to a separate file, and not embedded within the NFT itself. Ideally the license should be on-chain, but there are already a number of repository/sources of standard licenses.

Examples of existing NFT license:
- https://www.nftlicense.org/
- https://docs.solsea.io/licenses/license-scope-summary

And of course Creative Commons
- https://creativecommons.org/choose/

**Comment by Lux1n** [comment]
I support this initiative! It can only help the buyers and sellers in the art & general NFT market on DeSo to be aware of and have easy access to CC licensing standards. It increases buyer security in their investments, and protects artists while improving their reputation.

There could also be a way to lay out IP/commercial rights to piece once it's been bought.

**Comment by 0xNordian** [comment]
I support this idea. My only concern to not put the data on-chain is what happen if the licence change in the future and previously minted NFT point to the wrong license or any license at all? 

One possibility is to create a repository of licences on the chain with versions; in that way we should be able to avoid the described problem.

I'm not an expert in this field, so maybe I'm missing a piece of information that solves my concern, but that is the feedback that I can give at the moment.

Thanks for sharing 🤘
